src/upchunk.ts

```diff
@@ -290,20 +390,46 @@ export class UpChunk {
     this.success = false;
     this.nextChunkRangeStart = 0;
 
+    if (options.useFileSliceFallback) {
+      const readableStreamErrorCallback = (event: CustomEvent) => {
+        // In this case, assume the error is a result of file reading via ReadableStream.
+        // Retry using ChunkedFileIterable, which reads the file into memory instead
+        // of a stream.
+        if (this.chunkedIterable.error) {
+          console.warn(
```

Comment:
We "swallow the error" if we're falling back, but still log a warning for folks, just in case devs want some visibility here.